src/install-pnpm/run.ts

```diff
@@ -1,3 +1,4 @@
+import * as core from '@actions/core'
 import { spawn } from 'child_process'
 import { execPath } from 'process'
 import { join } from 'path'
```

Comment:
Since `path` is a built-in module, `import path from 'path'` would not increase bundle size, and since `join` is ambiguous, I suggest using `import path from 'path'` and `path.join`.